frontend/src/components/ChatBox/ChatBoxMessage/MessageBubble.tsx

```diff
@@ -36,26 +36,25 @@ function MessageBubble({
 
 	const messageAuthor =
 		message.type === CHAT_MESSAGE_TYPE.LEVEL_INFO
-			? 'Information message: '
+			? ''
 			: message.type === CHAT_MESSAGE_TYPE.USER
-			? 'You said: '
+			? 'You said:'
 			: message.type === CHAT_MESSAGE_TYPE.USER_TRANSFORMED
 			? 'Your message transformed by XML tagging: '
 			: message.type === CHAT_MESSAGE_TYPE.ERROR_MSG
-			? 'Error message: '
-			: message.type === CHAT_MESSAGE_TYPE.BOT
-			? 'ScottBrewBot said:  '
-			: 'ScottBrewBot said: ';
+			? 'Error message:'
+			: 'ScottBrewBot said:';
 
 	return (
 		// eslint-disable-next-line jsx-a11y/no-noninteractive-tabindex
-		<section className={className} tabIndex={0}>
-			{message.type === CHAT_MESSAGE_TYPE.USER_TRANSFORMED && (
-				<b>Transformed: </b>
-			)}
+		<section className={className} lang="en" tabIndex={0}>
 			{message.type === CHAT_MESSAGE_TYPE.LEVEL_INFO && (
-				<p className="header">Information</p>
+				<>
+					<p className="header">Information</p>
+					<span className="visually-hidden"> message: </span>
+				</>
 			)}
+			<span className="visually-hidden">{messageAuthor}</span>
```

Comment:
This is real pedantic, so you can ignore if you wish, but I'd prefer to only render the messageAuthor span if it's going to have a value.
i.e.
```typeescript
{message.type === CHAT_MESSAGE_TYPE.LEVEL_INFO: (
				<>
					<p className="header">Information</p>
					<span className="visually-hidden"> message: </span>
				</>
			) ? (<span className="visually-hidden">{messageAuthor}</span>)
}
```

Reply:
For consistency and clarity I kind of want to keep it as it is for now if that's okay?

Reply:
sure thing. I suppose clarity is subjective!